ontology/molecule.tsv

```diff
@@ -27559,3 +27559,4 @@ white-lipped tamarin MHC class I protein complex	white-lipped tamarin		class	equ
 white-lipped tamarin MHC class II protein complex	white-lipped tamarin		class	equivalent	MHC class II protein complex	white-lipped tamarin				
 yellow baboon MHC class I protein complex	yellow baboon		class	equivalent	MHC class I protein complex	yellow baboon				
 yellow baboon MHC class II protein complex	yellow baboon		class	equivalent	MHC class II protein complex	yellow baboon				
+HLA-DRA*01:01/DRB1*04:02 protein complex	HLA-DRA*01:01/DRB1*04:02	HLA-DRA1*0101/DRB1*0402	complete molecule	equivalent	MHC class II protein complex	human	HLA-DRA*01:01 chain	HLA-DRB1*04:02 chain		HLA-DR4 serotype
```

Comment:
This should be sorted alphabetically when you run `make update-tsv` (from the updated `mro.xlsx`). Did you update this sheet in a different way? You can also run `make sort` to do this without updating all TSVs.

Reply:
The only time I run anything on the command line is when I prepare `mro.xlsx` for @rvita. Here is the workflow I use:

- Go to the directory for MRO repo
- From master run `git pull`
- run `make mro.xlsx` to create an updated spreadsheet. 

After, I get the updated sheet from Randi I make the updates in DROID. Should I be running `make sort` after `make mro.xlsx` ?